fenix/app/src/main/java/org/mozilla/fenix/settings/SettingsFragment.kt

```diff
@@ -698,6 +717,16 @@ class SettingsFragment : PreferenceFragmentCompat() {
         }
     }
 
+    @VisibleForTesting
+    internal fun setupInstallAddonFromFilePreference(settings: Settings) {
+        with(requirePreference<Preference>(R.string.pref_key_install_local_addon)) {
+            isVisible = settings.showSecretDebugMenuThisSession &&
+                isNightlyOrDebugOrBeta()
```

Comment:
I think we don't need this condition, thanks for mentioning it during the standup 😅 We do want this feature in Release.

Reply:
removed!